app/components/ChatMessages.tsx

```diff
@@ -53,6 +55,40 @@ function ChatMessages() {
         const isFirst = index === 0;
         return (
           <Fragment key={message.id}>
+                        {displayDisclaimer && (
```

Comment:
```suggestion
                        {isFirst && displayDisclaimer && (
```
Only display disclaimer on first message - otherwise will show on every message like so:

<img width="350" alt="Image" src="https://github.com/user-attachments/assets/096293ab-d2f2-4ac1-b710-e7199630493b" />

In the initial implementation proposed, this also was only on the non-public version: https://github.com/wri/project-zeno-next/pull/258/commits/992210063c788c1e483de1410def49234e3b7df8

However, I think we could just proceed with this included on any version, and change the text or add/remove as needed